xmlrunner/runner.py

```diff
@@ -16,7 +16,7 @@ class XMLTestRunner(TextTestRunner):
     """
     def __init__(self, output='.', outsuffix=None, stream=sys.stderr,
                  descriptions=True, verbosity=1, elapsed_times=True,
-                 failfast=False, buffer=False, encoding=UTF8):
+                 failfast=False, buffer=False, encoding=UTF8, info_cls=_TestInfo):
```

Comment:
commit 887df0d2f9657e0236aec587dba70c43ae3fac58 should help with that part. Can you please rebase?